src/AddressRegistry/StreetName/Commands/RemoveAddressBecauseStreetNameWasRemoved.cs

```diff
@@ -0,0 +1,44 @@
+namespace AddressRegistry.StreetName.Commands
+{
+    using System;
+    using System.Collections.Generic;
+    using Be.Vlaanderen.Basisregisters.Generators.Guid;
+    using Be.Vlaanderen.Basisregisters.GrAr.Provenance;
+    using Be.Vlaanderen.Basisregisters.Utilities;
+
+    public class RemoveAddressBecauseStreetNameWasRemoved : IHasCommandProvenance
```

Comment:
Why this new command? I don't think it is ever used?

Reply:
Indeed, we changed the implementation and forgot to remove this.